pkgs/shells/fish/plugins/fish-you-should-use.nix

```
@@ -0,0 +1,26 @@
+{
+  lib,
+  buildFishPlugin,
+  fetchFromGitHub,
+  unstableGitUpdater,
+}:
+buildFishPlugin {
+  pname = "fish-you-should-use";
+  version = "unstable-2022-02-13";
```

Unstable versions should start with `0` (see [Package Naming](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming))

```suggestion
  version = "0-unstable-2022-02-13";
```